config.toml

```
@@ -18,6 +18,18 @@ allowed-github-orgs = [
     "rust-analyzer",
 ]
 
+allowed-domains = [
+    "https://(.*)rust-lang.org/?.*",
```

```suggestion
    "https://(.*)\\.rust-lang.org/?.*",
```
instead of all of this regex, would it make more sense to parse the URLs and check whether the host part ends with a list of allowed host suffixes?

---

That was the first approach that I considered, but then dropped when I came across the link to the GitHub issue and the `bit.ly` link. There are probably valid reasons to link to domains that we don't own, but in that case I would only want to allow specific URLs and not all of `bit.ly` or GitHub.